### PR TITLE
Project finished

### PR DESCRIPTION
**Pull request by webmarkyn**

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under review by Ahmed Mahfoudh TSE

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background: #eceef1;
+  height: 100%;
+  font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (stratospherique) [inline styles.css:10]: - I thing the height property is causing the superposition of your content on each other. Make it auto or just remove it :nerd_face: .

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work :clap: . However, some requirements needs to be implmented in order to get approval for this project. check below :arrow_down: 

#### Page structure

##### Sign In Button

* [ ] It has a Sign In button at the top right corner.
> It looks clipped from the bottom

##### Intuit products

* [ ] The products row is under the Intuit logo and above the form.
* [ ] It shows the Mint logo and text (logo on the left, text on the right).
* [ ] It shows the Quickbooks logo and text (logo on the left, text on the right).
* [ ] It shows the Turbotax logo and text (logo on the left, text on the right).

##### Footer

* [ ] It contains copyright and address information.
> Again this sits on top of the form because you have limited the body height to `100%`

#### Implementation details

###### Middle

* [ ] It has an `<input>` of type `email` and a label `Email address`.
* [ ] It has an `<input>` of type `tel` and a label `Phone`.
* [ ] It has an `<input>` of type `password` and a label `Password`.
> Use `<label for="id-of-input">` in order to link each label with it's input :nerd_face: 

### That's it guys :relaxed: . When finished implementing the above, kindly resubmit another code review request :pray:

**Review by Redvanisation — CHANGES_REQUESTED** [review]
### Hello @webmarkyn and @rkyankya 🌟 

_**Well done on this project, however there are still some things that need to be changed:**_

1. **Fix the README** file adding title and formatting it nicely, a link for reference: https://github.com/tundeiness/New-York-Times-clone/blob/master/README.md

2. **Validate the HTML markup** and fix all the errors: https://validator.w3.org/#validate_by_input

### Please resubmit a code review request after implementing the above changes. ✔️ 

### Good luck guys! 🍀

**Pull request merged**